app/views/exam_templates/_form.html.erb

```diff
@@ -3,6 +3,12 @@
     <%= f.error_messages%>
     <div class='template-pane'>
       <h4>
+        <%= link_to t('exam_templates.delete.all_general'), action: 'destroy_all' %>
+      </h4>
+      <h4>
+        <%= link_to t('exam_templates.delete.general'), action: 'destroy', id: exam_template.id %>
```

Comment:
This is not the normal Rails approach. See http://guides.rubyonrails.org/form_helpers.html#removing-objects.